deployment/v3/external/msg-gateway/install.sh

```diff
@@ -32,12 +33,12 @@ function msg_gateway() {
       read -p "Please enter the SMTP secret key " SECRET
   fi
 
-  kubectl -n $NS delete --ignore-not-found=true configmap email-gateway
-  kubectl -n $NS create configmap email-gateway --from-literal="email-smtp-host=$HOST" --from-literal="email-smtp-port=$PORT" --from-literal="email-smtp-username=$USER"
-  kubectl -n $NS delete --ignore-not-found=true secret email-gateway
-  kubectl -n $NS create secret generic email-gateway --from-literal="email-smtp-secret=$SECRET" --dry-run=client  -o yaml | kubectl apply -f -
+  kubectl -n $NS delete --ignore-not-found=true configmap msg-gateway
+  kubectl -n $NS create configmap msg-gateway --from-literal="email-smtp-host=$HOST" --from-literal="email-smtp-port=$EMAIL_PORT" --from-literal="sms-smtp-port=$SMS_PORT" --from-literal="email-smtp-username=$USER"
```

Comment:
sms-smtp port ?

Reply:
yes.. for email we are using 8025 for sms 8080 so i have updated like that ..